module/mailnews/mailfile.c

```diff
@@ -134,6 +134,7 @@ void MailNewMessage(int server_index, char *sender, int num_recipients,
    char *subject = "";
    char *ptr;
    int i, num;
```

Comment:
While we're at it, let's declare i inside the for loop on line 158 (and remove it here) so that we can't get into this buggy situation again.